Voycar.Api.Web.Tests.Integration/Users/Post_Register/Tests.cs

```diff
@@ -0,0 +1,327 @@
+namespace Voycar.Api.Web.Tests.Integration.Users.Post_Register;
+
+using Context;
+using Entities;
+using Microsoft.EntityFrameworkCore;
+using R = Features.Users.Endpoints.Post.Register;
+
+
+public sealed class State : StateFixture
+{
+    public Guid Id { get; set; }
+
+    public static Guid RoleId { get; set; }
+    public const string RoleName = "member";
+
+    public static Guid PlanId { get; set; }
+    public const string PlanName = "basic";
+
+    public R.Request Request { get; set; } = CreateValidRequest();
+
+
+    public static R.Request CreateValidRequest()
```

Comment:
Diese Methode auch nicht static, da sie dann ja auf ein Instanz-Attribute zugreift. Und zusätzlich die Methode bitte in die Testklasse. Hat eig. nichts mit dem State zu tun. 
Der State soll nur minmale Informationen, die zwischen Tests geteilt werden, speichern.

Reply:
ah ok. gut zu wissen